app/controllers/stories_controller.rb

```diff
@@ -368,7 +367,7 @@ def story_params
       :merge_story_short_id, :is_unavailable, :user_is_author, :tags_a => [],
     )
 
-    if @user.is_moderator?
+    if @user&.is_moderator?
```

Comment:
```suggestion
    if @user && @user.is_moderator?
```

We don't use `&.` in the codebase. Unfortunately rubocop does not offer a cop to prohibit it.

Reply:
no worries, done!